### PR TITLE
 [DOCS] Port over "How to host and share Data Docs on GCS" from RTD to Docusaurus 

### PR DESCRIPTION
**Pull request by cdkini**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- Port over `.rst` guide to `.md`
- GDOC-106/GDOC-190: https://superconductive.atlassian.net/browse/GDOC-190?atlOrigin=eyJpIjoiOGNkMmVhZDU4YmViNGUxNzllZTg0MTJjNzUxZmRmMTYiLCJwIjoiaiJ9
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-gcs.md [hunk 1]
```diff
@@ -1,5 +1,171 @@
...
+
+  We recommend placing it in your project directory, for example ``great_expectations/team_gcs_app``.
+
+  ```yaml
+  # app.yaml (make sure to use your own bucket name)
+
+  runtime: python37
+  env_variables:
+      CLOUD_STORAGE_BUCKET: my_org_data_docs
+  ```
```
Comment (cdkini) [inline docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-gcs.md]: Please take a look at how this (and other code blocks) are formatted in RTD. I tried to transition it over as best as I could but let me know if there's room for improvement with captions and line numbers: https://docs.greatexpectations.io/en/latest/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.html

**Review by cdkini — COMMENTED** [review]
Left a minor comment regarding porting over from RTD; please let me know your thoughts!

File: docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-gcs.md [hunk 2]
```diff
@@ -1,5 +1,171 @@
...
+      site_index_builder:
+        class_name: DefaultSiteIndexBuilder
+  ```
+
+7. **Build the GCS Data Docs site.**
+
+  Use the following CLI command: ``great_expectations --v3-api docs build --site-name gs_site``. If successful, the CLI will provide the object URL of the index page. Since the bucket is not public, this URL will be inaccessible. Rather, you will access the Data Docs site using the App Engine app configured above.
+
+  ```bash
+  > great_expectations --v3-api docs build --site-name gs_site
```
Comment (alexsherstinsky) [inline docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-gcs.md]: @cdkini Not sure here -- do we need the "greater-than" sign?  Was this supposed to be "a terminal/shell prompt" of sorts?  Thanks!

**Review by alexsherstinsky — CHANGES_REQUESTED** [review]
Small comment...

File: docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-gcs.md (hunk 2 above)
Comment (cdkini) [reply docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-gcs.md]: This was in RTD but I've updated per your comment!

**Comment by cdkini** [comment]
Resolved comment please re-review :)

**Review by alexsherstinsky — APPROVED** [review]
LGTM

**Pull request merged**